plutus-benchmark/script-contexts/src/PlutusBenchmark/V2/ScriptContexts.hs

```diff
@@ -139,3 +192,34 @@ mkScriptContextEqualityOverheadCode sc =
   in $$(PlutusTx.compile [|| scriptContextEqualityOverhead ||])
     `PlutusTx.unsafeApplyCode` PlutusTx.liftCodeDef sc
     `PlutusTx.unsafeApplyCode` PlutusTx.liftCodeDef d
+
+forwardWithStakeTrick :: BuiltinData -> BuiltinData -> ()
```

Comment:
Add some comments - it's hard to tell from the name - forwardWithStakeTrick - what it does.